Training/UI/Component/Listing/Notes/Column/AdminNameReceiver.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+declare(strict_types=1);
+
+namespace Codifi\Training\UI\Component\Listing\Notes\Column;
+
+use Magento\Ui\Component\Listing\Columns\Column;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\User\Model\ResourceModel\User\CollectionFactory;
+
+/**
+ * Class AdminNameReceiver
+ * @package Codifi\Training\UI\Component\Listing\Notes\Column
+ */
+class AdminNameReceiver extends Column
+{
+    /**
+     * Admin collection factory
+     *
+     * @var CollectionFactory
+     */
+    private $userCollectionFactory;
+
+    /**
+     * AdminNameReceiver constructor.
+     *
+     * @param ContextInterface $context
+     * @param UiComponentFactory $uiComponentFactory
+     * @param array $components
+     * @param array $data
+     * @param CollectionFactory $userCollectionFactory
+     */
+    public function __construct(
+        ContextInterface $context,
+        UiComponentFactory $uiComponentFactory,
+        CollectionFactory $userCollectionFactory,
+        array $components = [],
+        array $data = []
+    ) {
+        $this->userCollectionFactory = $userCollectionFactory;
+        parent::__construct(
+            $context,
+            $uiComponentFactory,
+            $components,
+            $data
+        );
+    }
+
+    /**
+     * Prepare Data Source
+     *
+     * @param array $dataSource
+     * @return array
+     */
+    public function prepareDataSource(array $dataSource): array
+    {
+        if (isset($dataSource['data']['items'])) {
+            $fieldName = $this->getData('name');
+            foreach ($dataSource['data']['items'] as & $item) {
+                if ($item[$fieldName] != '') {
```

Comment:
It's better to use `empty()` check or if you use comparison try to use `===` instead of `==` for check type of variables too